### PR TITLE
[feat] Navigation - Navbar and Sidebar

### PR DESCRIPTION
**Pull request by shivampandey0**
## Navigation Components

- Added Navbar component
- Added Sidebar component
- Added Fonts and `preconnect` for CSS

image

### CODE REVIEW

File: src/component/sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+} from 'react-icons/fa';
+
+export const Sidebar = () => {
+  return (
+    <aside className='drawer flex-column'>
+      <div className='drawer-nav'>
+        <nav className='drawer-list flex-column'>
+          <NavLink
+            to={'/'}
+            className={({ isActive }) =>
```
Comment (MaruthiPothuganti) [inline src/component/sidebar/Sidebar.jsx:18]: Toggling class with isActive prop is awesome. Learnt a new thing.

**Review by MaruthiPothuganti — COMMENTED** [review]
Clean code. Looks good to merge.

File: public/index.html [hunk 2]
```diff
@@ -24,7 +24,22 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-      <link rel="stylesheet" href="https://marvel-ui.vercel.app/css/marvel-ui.css">
+
+    <!-- Lato fonts -->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (GuptaEesh) [inline public/index.html:30]: Good use of pre-connect tag for faster CL loading.

**Review by GuptaEesh — COMMENTED** [review]
Good folder structure used.
Mentioned a comment.
Good to merge

**Review by shuklahimanshu81 — APPROVED** [review]
There is so much to learn from your code. Good use of conditionals to toggle classes. Code structure is clean. Good to merge.

**Pull request merged**